server/pbench/bin/index-pbench

```diff
@@ -863,15 +964,17 @@ class ToolData(object):
     def make_source(self):
         """Simple jump method to pick the write source generator based on the
         handler's prospectus."""
-        if not self.csv_files and not self.json_files:
-            # If we do not have any csv or json files for this tool, ignore it.
+        if not self.csv_files and not self.json_files and not self.stdout_files:
+            # If we do not have any files for this tool, ignore it.
             return
         if self.handler['@prospectus']['handling'] == 'unify':
             gen = self._make_source_unified()
         elif self.handler['@prospectus']['handling'] == 'individual':
             gen = self._make_source_individual()
         elif self.handler['@prospectus']['handling'] == 'json':
             gen = self._make_source_json()
+        elif self.handler['@prospectus']['handling'] == 'stdout':
```

Comment:
Can we consider renaming `stdout` to something closer to the contents we are targeting?  Perhaps, though kind of long: `periodic_timestamp_key_value`?